server/app/assets/javascripts/enumerator.ts

```diff
@@ -131,7 +138,14 @@ function removeExistingEnumeratorField(event: Event) {
 
   // Hide the field that was removed. We cannot remove it completely, as we need to
   // submit the input to maintain entity ordering.
-  const enumeratorFieldDiv = assertNotNull(removeButton.parentElement)
+  let enumeratorFieldDiv = assertNotNull(removeButton.parentElement)
```

Comment:
**suggestion (non-blocking):** It might be nice to put this in a helper function since you repeat this loop above.

Reply:
Yes, good call! I didn't originally because one used parentElement and the other used parentNode, but seems both work, so pulled it out into a helper!